src/lib/bootstrap.js

```diff
@@ -1,16 +1,16 @@
 import isFunction from 'lodash/isFunction';
+import assign from 'lodash/assign';
+import isObject from 'lodash/isObject';
```

Comment:
Why not `import _ from 'lodash'` from `import {assign, isObject} from 'lodash'`?

`assign` could replace by Object.assign or
```js
{
  ...originObject,
  newProperty: 'foo',
  anotherProperty: 'bar'
}
```

Reply:
`import {assign, isObject} from 'lodash' ` 👍